src/components/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -56,14 +57,15 @@ const AccordionItem: React.FC<AccordionItemProps> = forwardRef(
     }, [onClickAccordionCallback, onClick]);
 
     return (
-      <div ref={mergedRef} className={cx("accordion-item", className)} id={id}>
+      <div ref={mergedRef} className={className} id={id}>
         <ExpandCollapse
           iconSize={iconSize}
           id={`expand-collapse--${id}`}
           onClick={onClickCallback}
           open={open}
           title={title}
           hideBorder={hideBorder}
+          componentClassName={expandCollapseComponentClassName}
```

Comment:
no classname prpo?

Reply:
All good, `expand-collapse` was the one which was getting overriding 
![image](https://user-images.githubusercontent.com/104433616/236230359-a5a2d1a6-439e-4844-8691-75829766f538.png)